twentytwentyone-blocks/README.md

```diff
@@ -0,0 +1,47 @@
+# Twenty Twenty-One Blocks
+
+Ongoing development for the block-based version of Twenty Twenty-One, the default WordPress theme slated for 5.6. 
+
+This theme requires the Gutenberg Plugin to be installed, and the "Full Site Editing" experiment to be activated.
+
+For the non-block-based version of this theme, please [visit the Twenty Twenty-One repository](https://github.com/WordPress/twentytwentyone). 
+
+To contribute to TT1 development, please read the [contributor's guide](/CONTRIBUTING.md).
+
+## Installation
+
+1. In your admin panel, Install and Activate [the Gutenberg Plugin](https://wordpress.org/plugins/gutenberg/). 
+2. Visit the Gutenberg -> Experiments settings page and check the "Enable Full Site Editing" box. Save your changes. 
+3. Click the 'Code' button on [the main Theme Experiments Repository GitHub page](https://github.com/wordpress/theme-experiments).
+4. Unzip the resulting download. 
+4. Create a new zip file containing only the "twentytwentyone-blocks" folder. 
```

Comment:
Because @scruffian's template resolution fix did not make it in 9.2.1, you'll still get a "Template Part Not Found" error if you try to load the theme exists in a subdirectory.

But I don't think this should be a blocker for merging the PR.

Reply:
Cool, we can revise this README once that's merged in. 👍